### PR TITLE
Add local-imports.md to docs

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

File: docsite/docs/Docs/fal-cli/local-imports.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    └── my_utils
+        ├── custom_functions.py
+        └── message
+            └── slack.py
+```
+
+This will allow you to do local imports in `after.py`:
+
+```python
+from my_utils.message.slack import send_slack_message
```
Comment (chamini2) [inline docsite/docs/Docs/fal-cli/local-imports.md:41]: Question: would `from .my_utils.message.slack import ...` work too?

File: docsite/docs/Docs/fal-cli/local-imports.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+...
+send_slack_message(my_message)
+...
+```
+
+Also, within `slack.py` you can do relative imports:
+
+```python
+from ..custom_functions import format_string
```
[the remark this answers is not in the document]
Comment (mkam039) [reply docsite/docs/Docs/fal-cli/local-imports.md:50]: Yes!

File: docsite/docs/Docs/fal-cli/local-imports.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+...
+formatted = format_string(my_string)
+...
+```
+
+Note that in this example, `my_scripts_dir` is not itself loaded as a module, so a similar import wouldn't work from `custom_functions.py`.
```
[the remark this answers is not in the document]
Comment (mkam039) [reply docsite/docs/Docs/fal-cli/local-imports.md:56]:
```
from .message.slack import *
```
would work

File: docsite/docs/Docs/fal-cli/local-imports.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```
+fal run --scripts after.py
+```
+
+## Local imports in your fal scripts
+The directory containing fal scripts is temporarily added to `sys.path` during runtime. This means that you can import local modules in your scripts directory. Say we have the following structure in our scripts directory:
+
+```
+└── my_scripts_dir
+    ├── after.py
```
Comment (turbo1912) [inline docsite/docs/Docs/fal-cli/local-imports.md:31]: document how `custom_functions.py` can import `after.py`

File: docsite/docs/Docs/fal-cli/local-imports.md (hunk 1 above)
Comment (mkam039) [reply docsite/docs/Docs/fal-cli/local-imports.md:42]: No, that wouldn't work.

File: docsite/docs/Docs/fal-cli/local-imports.md (hunk 4 above)
Comment (mkam039) [reply docsite/docs/Docs/fal-cli/local-imports.md:31]: Done

**Review by chamini2 — APPROVED** [review]
Great

**Review by turbo1912 — APPROVED** [review]

**Pull request merged**